### PR TITLE
Add documentation for version-string

### PR DESCRIPTION
**Pull request by jesobreira**
node-rcedit was also suffering the lack of documentation for `version-string` mentioned on electron/rcedit#2

This PR adds to the README examples of usage of the object, as seen on [unindented/grunt-rcedit](https://github.com/unindented/grunt-rcedit/)

By the way, unindented/grunt-rcedit#1 has added a table to the README that lists every possible key and explains what it means. However it doesn't fit to me to tell whether this repo must bring that information or if users should find that information on official sources (i.e.: MSDN).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
...
+	'file-version': '1.0.0',
+	'product-version': '1.0.0',
+
+	'version-string': {
+		'CompanyName': 'Acme Inc.',
+		'FileDescription': 'A test file',
+		'LegalCopyright': 'Copyright (C) Acme Inc.',
+		'ProductName': 'My Test'
+	}
+
```
Comment (malept) [inline README.md]: unnecessary whitespace

File: README.md [hunk 2]
```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
 * `application-manifest` - String path to a local manifest file to use.
   See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191.aspx)
   for more details.
+ 
+`version-string` allows you to set the PE file meta keys, such as in the following example:
+
+```
+var options = {
+
```
Comment (malept) [inline README.md]: unnecessary whitespace

File: README.md [hunk 3]
```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
 * `application-manifest` - String path to a local manifest file to use.
   See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191.aspx)
   for more details.
+ 
+`version-string` allows you to set the PE file meta keys, such as in the following example:
+
+```
+var options = {
```
Comment (malept) [inline README.md]: This block should probably go in an "options example" section (I'm not entirely sure what the section should be titled though), since it's showing more than just an example of `version-string`.

File: README.md [hunk 4]
```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
 * `application-manifest` - String path to a local manifest file to use.
   See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191.aspx)
   for more details.
+ 
+`version-string` allows you to set the PE file meta keys, such as in the following example:
```
Comment (malept) [inline README.md]: This should have a link to the MSDN docs, like `application-manifest` does.

File: README.md [hunk 5]
```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
...
+
+```
```
Comment (kevinsawicki) [inline README.md]: Might want to make this `js` so it highlights.

```

```js

```

**Comment by ckerr** [comment]
Closing stale PRs. This PR has not addressed the last code review from over a year ago.

Please reopen or resubmit this PR after addressing the reviewer's comments. Thank you!
